chart/config/releases.yaml

```diff
@@ -18,6 +18,10 @@ releases:
     image:
       repository: ghcr.io/cfcontainerizationbot/kubecf-apps-dns
       tag: 0.1.0
+  kubecf_kubectl:
+    image:
+      repository: ghcr.io/cfcontainerizationbot/kubecf-kubectl
```

Comment:
I think I like leaving a comment here to point to `google/cloud-sdk` as an alternative image. Just to leave some breadcrumbs if anyone ever runs into the situation where `kubecf-kubectl` has been accidentally put into a private repo as well.

Reply:
confirmed